src/common/components/Popover.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {Component, ReactText, ReactNode} from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import Menu from 'react-native-material-menu';
+import PopoverItem from './PopoverItem';
+
+interface IPopoverProps {
+	options: Array<{
+		title: ReactText;
+		id: ReactText;
+		icon: ReactNode;
+		isStripped?: boolean;
+		isDisabled?: boolean;
+		onSelect: (item: any) => void;
+	}>;
+	style?: object;
+}
+
+export default class Popover extends Component<IPopoverProps> {
```

Comment:
pure component

Reply:
No, i use a children inside component.

Read this: https://blog.cloudboost.io/react-purecomponents-children-979e3da15ba8